src/table/cells/handle-cell-focus.js

```diff
@@ -80,3 +80,49 @@ export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement
     }
   }
 };
+
+const getMemoisedSrNotation = () => {
+  let prevSelectedCount = 0;
+
+  return ({ focusedCellCoord, rootElement, selState }) => {
+    if (focusedCellCoord[0] === 0) return '';
+
+    const [rowIdx, colIdx] = focusedCellCoord;
+    const rowElements = rootElement.getElementsByClassName('sn-table-row');
+    const cell = rowElements[rowIdx]?.getElementsByClassName('sn-table-cell')[colIdx];
+
+    const tCellContent = cell && cell.innerText;
+    const isCellSelected = cell && cell.classList.contains('selected');
+
+    let notation = '';
+
+    if (selState.rows.length) {
+      const isSingularSelection = selState.rows.length === 1;
+      const selectionNote = `There ${isSingularSelection ? 'is' : 'are'} ${selState.rows.length} selected value${
+        isSingularSelection ? '' : 's'
+      } currently.`;
+
+      if (prevSelectedCount < selState.rows.length) {
+        // if we select cell
+        notation += `Value is selected. ${selectionNote}`;
+      } else if (prevSelectedCount > selState.rows.length) {
+        // if we deselect cell
+        notation += `Value is deselected. ${selectionNote}`;
+      } else if (isCellSelected) {
+        // if we are in selection mode and move to selected cell
+        notation += `${tCellContent},value is selected.`;
```

Comment:
Wouldn't this be announced twice? since it will already read the cell content

Reply:
for some reason only **when we are in selection** state, chrome vox will not pick up the cell content when we changing the focus, but the NVDA is fine with it.
so the reason for adding cell content here was that and by adding it, chrome vox works fine and NVDA will convey it twice.

✅ So Since we are targeting NVDA as the main screen reader, i just removed it in order to prevent the double announcement issue.